packages/flutter_tools/lib/src/web/bootstrap.dart

```diff
@@ -2,6 +2,8 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
+import 'dart:io';
```

Comment:
Sorry if I missed it but is this new import used?

Reply:
No worries: `Platform.isWindows`.

Reply:
Ahh of course. I dislike the syntax highlighting of strings in this review UI. It's hard quickly to see the interpolation bits.